schemas/bedrock-authn-token-http.js

```diff
@@ -0,0 +1,170 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const postTokenQuery = {
```

Comment:
It looks like maybe that's old and we don't need it? If you can double check that `req.query` isn't used where this would be applied ... and any other place where `req.query` *is* used we have a query validator for every value that is pulled out of it, then this can be removed.

Reply:
Removed.